### ISSUE TITLE
Preference to distinguish between null and empty columns

### ISSUE THREAD

**Issue by jamesbassett** [super-csv/super-csv#13]
Super CSV reads both null columns and columns with empty string (2 quote characters) as `null`.  

Null columns (returned as `null`):

    col1,col2,col3
    ,,

Empty strings (also returned as `null`):

    col1,col2,col3
    "","",""

While this is usually the desired behaviour, some use cases need to be able to distinguish between `null` and empty. Add a preference to allow empty strings to be read as `""` (empty string).  Note: this applies no matter what the quote character is.

Originally raised as [Bug x](https://sourceforge.net/p/supercsv/bugs/52/) on SourceForge.

**Comment by kbilsted** [comment]
:+1:

**Comment by pietroaragona** [comment]
I would assign this issue to myself, but there is not a button to do this... Maybe the owner must add me?

**Comment by kbilsted** [comment]
Yes @pietroaragona  you need to be a member for that. I think your message here is enough. You are welcome on the team if you want (and have time) to solve bugs, help with questions on SO or do new development :)

**Comment by pietroaragona** [comment]
I used super csv in my last work with pleasure and now I'd like help a bit the project, I would be honored to be part of your team. I can start to solve some bugs and help with releases (I like DevOps area also)

**Comment by kbilsted** [comment]
Sounds nice :)

**Comment by pietroaragona** [comment]
@kbilsted  I'm working on this issue.
In order to avoid that more people work on the same issue, can you assigne it to me?

**Comment by kbilsted** [comment]
I can't assign you unless you join the csv team. I think you are safe having adressed you working on the item here.  You are certainly welcome to join the team after having done a bit of work or helping people with questions. Eg  on SO

**Comment by imkrmgn** [comment]
Nice to meet you. Everyone.

It is my first time to use github. Also, I am also talking in English for the first time.

I sent a PR #122 on a problem similar to this topic.
I am glad if you give us your opinion.

**Comment by sairahu** [comment]
Hi everyone,
